packages/sdk/python/src/latitude_sdk/reference.md

```
@@ -9181,3 +9181,583 @@ client.experiments.delete(
 </dl>
 </details>
 
+## Sessions
+<details><summary><code>client.sessions.<a href="src/latitude_sdk/sessions/client.py">list</a>(...) -> PaginatedSessions</code></summary>
+<dl>
+<dd>
+
+#### 📝 Description
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)